lib/ReactTags.js

```diff
@@ -243,18 +246,49 @@ class ReactTags extends React.Component {
     })
   }
 
+  isValid () {
+    return this.state.validationState
+  }
+
+  _getValidationState () {
+    const tl = this.props.tags.length
+
+    let validationState = true
+    if (this.props.minTags && tl < this.props.minTags) {
+      validationState = false
+    }
+
+    if (this.props.maxTags && tl > this.props.maxTags) {
+      validationState = false
+    }
+
+    return validationState
+  }
+
+  componentDidUpdate (prevProps) {
+    const validationState = this._getValidationState()
+
+    if (this.state.validationState !== validationState) {
+      this.setState({
+        validationState
+      })
+    }
+  }
```

Comment:
I don't think we need to store this in state, because the implementation of `_getValidationState()` is so simple I think it's safe to call within `render()`.